### ISSUE TITLE
What framework to use?? 

### ISSUE THREAD

**Issue by Twanawebtech** [IBM-Bluemix/finance-trade#19]
We talked about where if we should use a framework or not for FinTrade. 

**Options:**
- [ ] Use no any framework. The issue with this is that we will end up writing a lot of Jquery/AJAX code where it could have been done more easily with any of the below frameworks. 
- [ ] React.js 
- [x] Angular 1  
- [ ] Vue.js

If I select one, I think I will go with Angular 1 because I am comfortable coding with Angular 1  and I know it well. At any case, I am going to explore Vue.js as per @VidyasagarMSC suggestion.

**Comment by rvennam** [comment]
Vue is definitely the new hotness, but my suggestion would be to stick with what you know. Angular is still very popular and is more than capable of doing what you need. just my 2 cents.

**Comment by VidyasagarMSC** [comment]
The reason we started exploring VueJS is 

- React adopting new architecture called Fiber and if backward compatibilty is not an option.
- AngularJS (v1) is already a kind of Obsolete with the release of Angular 2 & 4 - using Typescript which is a new learning for both of us. 
- VueJS released a new version recently (so can't expect a major release soon) similar to React (Components & JSX syntax) & Angular (Directives). Also, no sample built using this yet. 

I am game for any framework/library unless it serves our cause and improves code readability.

**Comment by Twanawebtech** [comment]
@VidyasagarMSC I agree with points noted above. 
I have played with VeuJS, and it's nice, easy to pick up and also a framework that learned a lot from the mistakes that Angular and ReactJS made. 

Another reason why I started to get soft on using VeuJS is the fact that we don't have any real VeuJS applications, **this would make a great Blog post** that could attract many VeuJS developers. 

Let's discuss more on our daily scrum call.
